src/protocols/secure_channel/CASESession.cpp

```diff
@@ -826,7 +825,7 @@ CHIP_ERROR CASESession::HandleSigma2(System::PacketBufferHandle && msg)
     SuccessOrExit(err = decryptedDataTlvReader.GetBytes(mResumptionId, static_cast<uint32_t>(sizeof(mResumptionId))));
 
     // Retrieve peer CASE Authenticated Tags (CATs) from peer's NOC.
-    Credentials::CATValues peerCATs;
+    CATValues peerCATs;
```

Comment:
Could these locals all be set to `kUndefinedCATs`?

Reply:
I suggested CATValues should have constructor/initialization to zero out its entries to undefined. (We're always going to want to do that.)